app/policies/item_request_link_policy.rb

```diff
@@ -5,6 +5,8 @@ def initialize(item:)
   end
 
   def show?
+    return folio_pageable? if item.folio_item?
```

Comment:
This doesn't match my intuition about when we show item-level request links. I'm not sure it's actually wrong, but whoever reviews it should take this out for a good spin.

I believe we show item-level request links when:
- there isn't a location-level request link (which.. is kinda the opposite of `folio_pageable?`, hence my confusion)
- when we know from the it's current location that the item is pageable without even needing to look at its current availability (because it's in some state that we know isn't temporary but the item is unavailable).

I don't know if the circ rules adequately capture that latter case, and it seems like it'd more likely be a `Hold` request type (because something `Page`-able seems like it's likely pageable at the item level?).. but it's possible it all works out in the end?